k8s/templates/job-backup.yaml

```diff
@@ -0,0 +1,58 @@
+{{- if .Values.backup.enabled -}}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "chart.fullname" . }}-backup
+  labels:
+    app.kubernetes.io/name: {{ include "chart.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}  
+spec:
+  schedule: "{{ .Values.backup.schedule }}"
+  successfulJobsHistoryLimit: {{ .Values.backup.successfulJobsHistoryLimit }}
+  failedJobsHistoryLimit: {{ .Values.backup.failedJobsHistoryLimit }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: lifemonitor-backup
+            image: {{ include "chart.lifemonitor.image" . }}
+            imagePullPolicy: {{ .Values.lifemonitor.imagePullPolicy }}
+            command: ["/bin/sh","-c"]
+            args:        
+              - base_path="/var/data/backup" ;
```

Comment:
perhaps this code would be easier to maintain if placed in a script in the lifemonitor image that is used to execute this job.

Reply:
Script replaced by `lm-admin backup` command (https://github.com/crs4/life_monitor/pull/210/commits/ee8401ae6fe6b5b7cef96aca49977668c57e77c3)